srtcore/core.h

```diff
@@ -97,22 +97,23 @@ enum AckDataItem
     ACKD_BUFFERLEFT = 3,
     ACKD_TOTAL_SIZE_SMALL = 4,
 
-    // Extra fields existing in UDT (not always sent)
-
-    ACKD_RCVSPEED = 4,   // length would be 16
+    // Extra fields existing in UDT (not always sent).
+    ACKD_RCVSPEED = 4,  // length = 16
     ACKD_BANDWIDTH = 5,
-    ACKD_TOTAL_SIZE_UDTBASE = 6, // length = 24
-    // Extra stats for SRT
-
+    ACKD_TOTAL_SIZE_UDTBASE = 6,  // length = 24
+    
+    // Extra stats for SRT.
     ACKD_RCVRATE = 6,
-    ACKD_TOTAL_SIZE_VER101 = 7, // length = 28
-    ACKD_XMRATE = 7, // XXX This is a weird compat stuff. Version 1.1.3 defines it as ACKD_BANDWIDTH*m_iMaxSRTPayloadSize when set. Never got.
-                     // XXX NOTE: field number 7 may be used for something in future, need to confirm destruction of all !compat 1.0.2 version
-
-    ACKD_TOTAL_SIZE_VER102 = 8, // 32
-// FEATURE BLOCKED. Probably not to be restored.
-//  ACKD_ACKBITMAP = 8,
-    ACKD_TOTAL_SIZE = ACKD_TOTAL_SIZE_VER102 // length = 32 (or more)
+    ACKD_TOTAL_SIZE_VER101 = 7,  // length = 28
+    // XXX This is a weird compat stuff. Version 1.1.3 defines it as
```

Comment:
@maxsharabayko Please provide me better explanation for this TODO section, I'll update appropriately.